packages.html

```diff
@@ -0,0 +1,334 @@
+                <p class="repoDescription">Initial code for studying ToxCast data in relation to measured.</p>
+                <p class="underDevelopment">Research</p>
+            </div>
+          </a> 
+          
+          <a href="https://github.com/USGS-R/loadflex" target="_blank">
+            <div class="repos researchColor">
+            	<h2>loadflex</h2>
+                <p class="repoDescription">Models and Tools for Watershed Flux Estimates.</p>
+                <p class="underDevelopment">Research</p>
+            </div>
+          </a>
+        
+      </div>
+  </div>
+      
+              
+              <h3>Archive</h3>
+<p>A package with the designation “Archive Research” has several key features:</p>
+
+<div id="plain">
+<ul>
+  <li>Ideally, the development of the active research package was completed with the publishing of a scientific, peer-reviewed journal or report</li>
+  <li>A reproducible work-flow is included either within a vignette or via the final product (such as the journal article)</li>
+  <li>A message on the README indicates the archive status.</li>
+  <li>There are no official expectations that bug fixes and R version upgrades will happen.</li>
+  <li>The package maintainer should have an active email.</li>
+  <li>The package version should be updated to v1.0.0 once a stable version is released.</li>
+</ul>
+</div>
+
+<p>
+It should be clearly communicated that the package has a well-documented final product. An archived research R-package can certainly maintain development such as bug fixes and required changes to R version updates. Maintaining the package and interacting with users can be very beneficial for professional development. However, it is important to communicate the official level of support for future users. See <code>?packageStartupMessage</code> to learn how to add a message to that gets displayed whenever a user loads the package.
+</p>
+
+<h3>Example Archive Research Packages</h3>
+  <div id="wrapper" class="grid clearfix">
+      <div id="repoContainer">
+      <a href="https://github.com/USGS-R/EGRET" target="_blank"> 
+        <div class="repos researchColor">
+        	<h2>EGRET</h2>
+            <p class="repoDescription">An R-package for the analysis of long term changes in water quality and streamflow</p>
+            <p class="underDevelopment">Research</p>
+        </div>
+      </a>
+      <a href="https://github.com/USGS-R/EGRETci" target="_blank">
+        <div class="repos researchColor">
+        	<h2>EGRETci</h2>
+            <p class="repoDescription">A bootstrap method for estimating uncertainty of water quality trends.</p>
+            <p class="underDevelopment">Research</p>
+        </div>
+    	</a>           
+        
+        
+    </div>
+  </div>
+  
+  <h3>Exploratory</h3>
+  <p>These packages should remain on personal github repositories or local computers. These packages offer value to only a small number of people for a limited time.</p>
```

Comment:
especially if Exploratory becomes the first item under Research, add a first sentence relating these packages to the larger  category - something like,
`Package development may include an exploratory phase when it's unclear whether an idea will ever become an active, publicly supported package. These exploratory packages should remain...`